README.md

```diff
@@ -5,32 +5,19 @@
 [![Release](https://img.shields.io/github/v/release/mattermost/mattermost-plugin-zoom)](https://github.com/mattermost/mattermost-plugin-zoom/releases/latest)
 [![HW](https://img.shields.io/github/issues/mattermost/mattermost-plugin-zoom/Up%20For%20Grabs?color=dark%20green&label=Help%20Wanted)](https://github.com/mattermost/mattermost-plugin-zoom/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22Up+For+Grabs%22+label%3A%22Help+Wanted%22)
 
-
 **Maintainer:** [@larkox](https://github.com/larkox)
 **Co-Maintainer:** [@mickmister](https://github.com/mickmister)
 
-Start and join voice calls, video calls and use screen sharing with your team members via Zoom.
+Start and join voice calls, video calls and use screen sharing with your team members via Zoom. Check our [GitBook](https://mattermost.gitbook.io/plugin-zoom/) for further documentation about this plugin.
+
+## Usage and Set Up Guide
```

Comment:
Does this title really apply any longer?

Reply:
Nope, good point.